### DISCUSSION TITLE
Simplified session code

### DISCUSSION THREAD

**Discussion by bug-or-feature** [ig-python/ig-markets-api-python-library#317] [Ideas]
I'd like to simplify the session code, its way too complicated (I think).

Does anyone use the [CachedSession stuff](https://trading-ig.readthedocs.io/en/latest/rest.html#cache-queries-requests-cache)? I'd like to remove it completely.

**Comment by galburn** [comment]
I'm doing my own caching in Redis, so not using CachedSession.

**Comment by wookiesh** [comment]
Hello, yes I do, creating the session at IGservice level. 
I handle some more caching using MongoDB (telethon etc) but as it is backed in, that's nice.
